content/about/index.md

```diff
@@ -0,0 +1,195 @@
++++
+title = "About Matrix"
++++
+
+## Governance
+
+The evolution of Matrix is managed through an open governance process, looked
+after by The Matrix.org Foundation - ‡ non-profit UK Community Interest
+Company, incorporated to act as the neutral guardian of the standard on behalf
+of the whole Matrix community.
+
+The Foundation defines the manifesto, mission and values of the project, the
+open governance process that determines how the specification develops, and
+provides a safety-net to ensure the project stays independent and true to its
+goals. The constitution of the project is defined in the Foundation’s legal
+Articles of Association and Rules, and is enforced by the Guardians of Matrix:
+the directors of the Foundation appointed to provide a balance of independent
+experts and the founding Matrix team to ensure the project stays on track.
+
+### The Guardians
+
+The Guardians are the legal directors of the non-profit Foundation, and are
+responsible for ensuring that the Foundation (and by extension the Spec Core
+Team) keeps on mission and neutrally protects the development of Matrix.
+Guardians are typically independent of the commercial Matrix ecosystem and may
+even not be members of today’s Matrix community, but are deeply aligned with
+the mission of the project. Guardians are selected to be respected and trusted
+by the wider community to uphold the guiding principles of the Foundation and
+keep the other Guardians honest.
+
+In alphabetical order:
+{{ guardians() }}
+
+### Legal Details
+
+* The Matrix.org Foundation C.I.C is registered in the UK as Company 
+  [#11648710](https://beta.companieshouse.gov.uk/company/11648710)
+* The official articles of association of the Foundation may be
+  downloaded [here](/media/2019-06-10%20-%20Matrix.org%20Foundation%20CIC%20Articles%20of%20Association.pdf)
+* The official Rules for the Foundation may be downloaded [here](/media/2019-06-10%20-%20Matrix.org%20Foundation%20CIC%20Rules.pdf)
+
+
```

Comment:
We don't have double empty lines before other headings.

```suggestion
```

Reply:
(This probably is a leftover from gatsby. There all the mdx files have this too which is a little annoying :D)